db/seeds.rb

```diff
@@ -1,7 +1,9 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rake db:seed (or created alongside the db with db:setup).
-#
-# Examples:
-#
-#   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
-#   Mayor.create(name: 'Emanuel', city: cities.first)
+20.times do
+  User.create(
+    email: Faker::Internet.email,
+    username: Faker::Name.first_name,
+    password: Faker::Internet.password(8),
+    phone_number: Faker::Number.number(10),
+    gender: 'Male'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.